Fix Translations in strings and keys

chirag-jn opened the pull request:
Fixes #430 
Extends #432

gopikaharikumar committed:
Removed the hard coded strings

Requested changes

gopikaharikumar committed:
Merge branch 'beta' of https://github.com/amahi/android into hard

cpg commented:
Anything that is not translatable should really not be in any strings file, no?

e.g. `DownloadFinished`, or `INTENT_FILE` or `auto_upload_server`

correct?

chirag-jn committed:
Merge commit 'refs/pull/432/head' of https://github.com/amahi/android into hard-coded

chirag-jn committed:
TV App Name as intranslatable

chirag-jn committed:
remove duplicate intro_title_4

chirag-jn committed:
remove extra intent strings, introduce constants class

chirag-jn committed:
remove preference string

chirag-jn committed:
Remove Connection string

chirag-jn committed:
Remove Connection string

chirag-jn committed:
Add Constants
MD5
imageMediaType
videoVLCLibArg
emailTextPlainType
prepare, progress dialog fragment tag
download start, progress, finish, success extra, progress extra intent
shares path loc name

chirag-jn committed:
remove extra strings from hindi, malyalam

chirag-jn committed:
remove ellipsis

chirag-jn committed:
Add Introducing Cast

chirag-jn commented:
Created a Constants class as recommended by @codedentwickler in #432 
Creating a `static final` variable doesn't cause any type of memory leak when compared to `static` variables which are potential points of memory leaks.

chirag-jn committed:
Add Copyright in Constants.java
